control-plane/tests/test_service_restore_backup_codec.py

```
@@ -0,0 +1,834 @@
+                    ValueError,
+                    "restore payload record count for 'recommendation' must be an integer",
+                ):
+                    restored_service.restore_authoritative_record_chain_backup(
+                        invalid_backup
+                    )
+                self._assert_authoritative_store_empty(restored_store)
+
+    def test_service_phase21_restore_rejects_v2_backup_without_recommendation_family(
+        self,
+    ) -> None:
+        _store, service, promoted_case, _evidence_id, _reviewed_at = (
```

Ruff (0.15.12)

[warning] Unpacked variable `promoted_case` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)